components/expenses/PayExpenseModal.js

```diff
@@ -113,6 +115,9 @@ const PayExpenseModal = ({ onClose, onSubmit, expense, collective, host, error }
   const formik = useFormik({ initialValues, validate, onSubmit });
   const hasManualPayment = payoutMethodType === PayoutMethodType.OTHER || formik.values.forceManual;
   const payoutMethodLabel = getPayoutLabel(intl, payoutMethodType);
+  const hasBankInfoWithoutWise = host => {
+    return payoutMethodType === PayoutMethodType.BANK_ACCOUNT && host.transferwise === null;
+  };
```

Comment:
Since this is called with the same variable, I suggest evaluating only once:
```suggestion
  const hasBankInfoWithoutWise = payoutMethodType === PayoutMethodType.BANK_ACCOUNT && host.transferwise === null;
```